src/config/modelProviders/groq.ts

```diff
@@ -3,6 +3,27 @@ import { ModelProviderCard } from '@/types/llm';
 // ref https://console.groq.com/docs/models
 const Groq: ModelProviderCard = {
   chatModels: [
+    {
+      displayName: 'LLaMA3.1 405B',
+      enabled: true,
```

Comment:
Early API access to Llama 3.1 405B is currently only available to paying customers, So I think it can be disabled by default.